doc/changes/1961.bugfix

```diff
@@ -0,0 +1 @@
+adaptive time steps for integration of floquet rates
```

Comment:
```suggestion
Change floquet_master_equation_rates(...) to use an adaptive number of time steps scaled by the number of sidebands, kmax.
```

Reply:
Could you also fix the docstring for `kmax` (it's currently named `k_max`) and add a note to it to mention how the number of integration steps is scaled?